packages/smart-forms-renderer/src/stories/itemTypes/Attachment.stories.tsx

```diff
@@ -31,9 +35,49 @@ export default meta;
 type Story = StoryObj<typeof meta>;
 
 // More on writing stories with args: https://storybook.js.org/docs/react/writing-stories/args
+const targetlinkId = 'file-attachment'
+const targetText = 'File Attachment'
+
+const qAttachmentBasic = questionnaireFactory([{
+  linkId: targetlinkId,
+  type: 'attachment',
+  repeats: false,
+  text: targetText
+}])
+
+const url = 'http://world_of_warcraft.com'
+const name = 'Vladimir'
+const fileName = 'foo.png'
+const type = "image/png"
+
 
 export const AttachmentBasic: Story = {
   args: {
     questionnaire: qAttachmentBasic
+  },
+  play: async ({ canvasElement }) => {
+    await inputFile(
```

Comment:
ACT
Fill input (select, choose, input...)
Assert
Check answers in store (do not check visual state using canvas!)


ACT
Clean 
ASSERT
Wait (if needed)
Check store
Check visual state (canvas)